src/code/providers/document-store-share-provider.coffee

```diff
@@ -92,5 +92,7 @@ class DocumentStoreShareProvider
         error: (jqXHR) ->
           docName = metadata?.filename or 'document'
           callback "Unable to share '#{docName}'"
+    else
+      callback "Unable to unshare '#{docName}'"
```

Comment:
If there isn't already, there should be a technical debt story for moving all of the hard-coded CFM error messages into the strings file for future translatability.